### DISCUSSION TITLE
How can i get a player object from the username of that player?

### DISCUSSION THREAD

**Discussion by LazyGamer1111** [Minestom/Minestom#1890] [Help]
I want to send a player to another instance from the name of the player, but i cant get the player object from the name. How can i do this?

**Comment by nartsisss** [comment]
You should use [ConnectionManager#getPlayer](https://javadoc.minestom.net/net/minestom/server/network/ConnectionManager.html#getPlayer(java.lang.String)) to find player by specific name:
```java
var player = MinecraftServer.getConnectionManager().getPlayer(name);
```
or [ConnectionManager#findPlayer](https://javadoc.minestom.net/net/minestom/server/network/ConnectionManager.html#findPlayer(java.lang.String)) to find the closest player matching a given name:
```java
var player = MinecraftServer.getConnectionManager().findPlayer(name);
```
